pkg/ccl/importccl/read_import_proc.go

```diff
@@ -283,18 +277,18 @@ func (c *rowConverter) row(fileIndex int32, rowIndex int64) error {
 	}
 	// If our batch is full, flush it and start a new one.
 	if len(c.kvBatch) >= kvBatchSize {
-		if err := c.sendBatch(); err != nil {
+		if err := c.sendBatch(ctx); err != nil {
 			return err
 		}
 	}
 	return nil
 }
 
-func (c *rowConverter) sendBatch() error {
+func (c *rowConverter) sendBatch(ctx context.Context) error {
 	select {
 	case c.kvCh <- c.kvBatch:
-	case <-c.done:
-		return c.ctx.Err()
+	case <-ctx.Done():
```

Comment:
we think the lack of done caching is fine? i guess per batch isn't too bad.

Reply:
yeah, that's my guess. if a profile comes along later that says i'm wrong, we'll add some done caching thing then, but i doubt it it is on the same scale as the io and parsing allocs we do in import.